sources/AIP132-Zaptos/proposal/0-consensus-config.move

```diff
@@ -0,0 +1,49 @@
+// Script hash: d89c3666 
```

Comment:
could you please follow our naming scheme and use the date of the proposal creation for the folder name?